src/lib.rs

```diff
@@ -102,6 +102,8 @@ pub type Result<T> = ::std::result::Result<T, Error>;
 
 pub mod abi;
 pub mod api;
+
+#[cfg(all(feature = "fusedev", feature = "virtiofs", not(feature = "macfuse")))]
```

Comment:
why passthrough doesn't work on macos?

Reply:
passthrough has many syscall, It's very hard to port.